partner/pmp-partner-service/src/main/java/io/mosip/pmp/partner/controller/PartnerServiceController.java

```diff
@@ -325,4 +331,28 @@
 		response.setResponse(partnerService.getPartnerCertificate(requestDto));
 		return response;
     }	
+	
+	@ResponseFilter
+	@PostMapping("/searchPartner")
```

Comment:
1. Can we change api name partner/search
2. Enable roles

Reply:
Resolved